scripts/zones/Bastok_Mines/npcs/Christina.lua

```diff
@@ -102,7 +102,7 @@ entity.onEventFinish = function(player, csid, option)
                 player:setCharVar("SmilebringersFameReset", 1)
             end
         else
-            player:messageSpecial(ID.text.ITEM_CANNOT_BE_OBTAINED)
+            player:messageSpecial(ID.text.ITEM_CANNOT_BE_OBTAINED, 1742)
```

Comment:
Want to convert these to xi.items.ITEM_NAME?

Reply:
Sure thing! Will update shortly.